elements-code-tutorial/easy-run-code.md

```diff
@@ -230,9 +230,10 @@ e1-cli generatetoaddress 1 $ADDRGEN1
 sleep 10
 e2-cli listissuances
 
-IADDR=$(e1-cli gettransaction $ITXID | jq -r '.details[0].address')
-
-e2-cli importaddress $IADDR
+ISSUE_RAW_TX=$(e2-cli getrawtransaction $ITXID 1)
```

Comment:
As opposed to how it was before, this requires node2 to have the txindex enabled. While before you asked the address from node1 who has it in its wallet. Why is this change required perse?

Reply:
It was for a scenario where a second node wants to see an issuance. So e1 issues an asset and sends the reissuance token to e2 - e2 cannot reissue because it is not aware of the issuance so an address needs importing from the issuance tx. e2's user can look up the tx on the block explorer's assets list page and doesn't need to know the address issued to but yes, this approach required tx_index=1.

Perhaps I can leave it as it was and just add the 'use a txid to get an address' as a commented out option?

Reply:
Hmm, depends on your scenario. If the scenario is "user1 sends one of the issuance tx addresses to user2", then use `e1-cli gettransaction`. If the scenario is "user1 sends the issuance txid and user2 looks for an address, then yeah `e2-cli getrawtransaction` makes sense. Perhaps a note "if txindex is not enabled, an address can be found by entering the txid in a block explorer", but it's good as is.

Reply:
have added mention of need for index=1 in conf file for this to work and reinstated use of previous code and offered this as an option instead.